packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

```diff
@@ -675,6 +699,9 @@ export class AtomicCommerceSearchBox
     const isDisabled = this.isSearchDisabledForEndUser(
       this.searchBoxState.value
     );
+    if (!this.suggestionManager.suggestions.length) {
+      this.registerSearchboxSuggestionEvents();
+    }
```

Comment:
`willLoad`is too early, as all the `willLoad` aren't complete
`didLoad` is too late, as the first render did happen, and no suggestions were registered... thus, doing it during the render function of the lifecycle.